docs/elements/examples/shadcn-ui.mdx

```diff
@@ -506,3 +549,48 @@ export default function SignInPage() {
 }
 ```
 </CodeBlockDemo>
+
+
+## OTP Input
+
+The following example demonstrates how to make a one-time password (OTP) input with Clerk Elements.
+<CodeBlockDemo demos={[
```

Comment:
I tried doing this example but got the error "Error: You used a hook from "ActorProvider" but it's not inside a <ActorProvider> component." Any idea what I'm doing wrong?

Here's my code:

```
"use client";
import * as Clerk from "@clerk/elements/common";
import { cn } from "@/lib/utils";

export default function SignInPage() {
  return (
    <Clerk.Input
      type="otp"
      className="flex justify-center has-[:disabled]:opacity-50"
      autoSubmit
      render={({ value, status }) => {
        return (
          <div
            data-status={status}
            className={cn(
              "relative flex size-10 items-center justify-center border-y border-r border-input text-sm transition-all first:rounded-l-md first:border-l last:rounded-r-md",
              {
                "z-10 ring-2 ring-ring ring-offset-background":
                  status === "cursor" || status === "selected",
              }
            )}
          >
            {value}
            {status === "cursor" && (
              <div className="pointer-events-none absolute inset-0 flex items-center justify-center">
                <div className="animate-caret-blink h-4 w-px bg-foreground duration-1000" />
              </div>
            )}
          </div>
        );
      }}
    />
  );
}
```

Reply:
You will need it within a step, the first sign-in example has it within the step that is needed for one time code. This OTP example, is stripped back just to demonstrate how the otp input can be built, similar to the primitive OTP example code block https://clerk.com/docs/elements/examples/primitives

Reply:
Okay, thanks. I'll edit the sentence I added to give this context for readers 👍

Reply:
Sounds good, thank you!